.github/workflows/tests.yml

```
@@ -0,0 +1,38 @@
+name: tests
```

can we also change the name of the file to `ci.yml`? 
```suggestion
name: CI
```